dotnet-engine/UnleashEngine/IFFIAccess.cs

```diff
@@ -0,0 +1,21 @@
+namespace Unleash;
+
+internal interface IFFIAccess {
```

Comment:
Can we try golf this together next week? We should try a bit to get this better

Is it a blocker to leave this open and work against a dev version on the SDK side?

Reply:
Absolutely! The intention behind this is to have something that works well but at the same time gets the conversation going